gobblin-runtime/src/main/java/gobblin/runtime/JobContext.java

```diff
@@ -96,6 +99,13 @@ public JobContext(Properties jobProps, Logger logger) throws Exception {
         previousJobState.getPropAsInt(ConfigurationKeys.JOB_FAILURES_KEY, 0));
 
     if (GobblinMetrics.isEnabled(jobProps)) {
+      Configuration configuration = new Configuration();
+      String flowId = configuration.get("azkaban.flow.flowid");
```

Comment:
1. Defined a constant for azkaban.flow.flowid
2. Moved tag names to gobblin-core/src/main/java/gobblin/metrics/TagNames.java